conans/cli/commands/remote.py

```diff
@@ -0,0 +1,101 @@
+import json
+
+from conans.cli.command import conan_command, conan_subcommand, OnceArgument
+
+
+def output_remote_list_json(info, out):
+    myjson = json.dumps(info, indent=4)
+    out.writeln(myjson)
+
+
+def output_remote_list_cli(info, out):
+    for remote_info in info:
+        output_str = "{}: {} [Verify SSL: {}, Enabled: {}]".format(remote_info["name"],
+                                                                   remote_info["url"],
+                                                                   remote_info["verify"],
+                                                                   remote_info.get("enabled", True))
```

Comment:
Conan 2.0, we can assume that we always have the ``enabled`` attribute, right? It was this way to introduce it as not breaking.